src/styles/components/form/fields/_image.scss

```diff
@@ -9,6 +9,6 @@ $component-name: form-fields-image;
     background-position: center;
     background-repeat: no-repeat;
     width: 100%;
-    height: 30vh;
+    max-height: 30vh;
```

Comment:
I think this breaks the edit view?

Reply:
Looks like this:
![screenshot 2018-11-07 at 11 31 35](https://user-images.githubusercontent.com/776599/48129276-e7f3b580-e280-11e8-83ce-77547e9930d2.png)

Should look like this:
![screenshot 2018-11-07 at 11 31 53](https://user-images.githubusercontent.com/776599/48129277-e7f3b580-e280-11e8-8a14-d5b4c2da743f.png)

Reply:
Thanks. Not sure why I did this...something to do with the sociogram prompt, but reverting it doesn't cause a visible issue.